payments/utils.py

```diff
@@ -22,13 +19,13 @@ def rounded(func):
     @wraps(func)
     def wrapped(*args, **kwargs):
         value = func(*args, **kwargs)
-        return round_price(value)
+        return util_rounded(value, decimals=2)
```

Comment:
are you passing `decimals` explicitly to stress that we will have 2 decimal points in the result? since `2` is default for `decimals` anyway

Reply:
🤔 good point